draft-wang-ppm-dap-taskprov.md

```diff
@@ -167,9 +167,9 @@ selection. It is defined as follows:
 
 ~~~
 struct {
-    QueryType query_type;         /* Defined in I-D.draft-ietf-ppm-dap-02 */
-    Duration time_precision;      /* Defined in I-D.draft-ietf-ppm-dap-02 */
-    uint16 max_batch_query_count; /* Defined in I-D.draft-ietf-ppm-dap-02 */
+    QueryType query_type;         /* I-D.draft-ietf-ppm-dap-02 */
+    Duration time_precision;      /* I-D.draft-ietf-ppm-dap-02 */
+    uint16 max_batch_query_count; /* I-D.draft-ietf-ppm-dap-02 */
```

Comment:
Reviewer note: This change and the ones below it ensure that the "art" (the stuff between the "~~~") all fits in 72  columns. This improves rendering to text and html.